code/game/objects/items/grenades/plastic.dm

```diff
@@ -199,41 +199,7 @@
 	else
 		return ..()
 
-/obj/item/grenade/plastic/c4/attack_self(mob/user)
-	var/newtime = input(usr, "Please set the timer.", "Timer", 10) as num
-	if(user.get_active_held_item() == src)
-		newtime = CLAMP(newtime, 10, 60000)
-		timer = newtime
-		to_chat(user, "Timer set for [timer] seconds.")
-
-/obj/item/grenade/plastic/c4/afterattack(atom/movable/AM, mob/user, flag)
```

Comment:
????

Reply:
the afterattack and attack_self for /plastic and /plastic/c4 is functionally the same so I removed them. I tested to see if there was a difference in game that i didn't notice and all I could find was that you could plant a c4 on yourself but I didn't think that was really an issue. If you want I can prevent that from happening.